indexers/vector/FaissIndexer/__init__.py

```diff
@@ -35,6 +37,7 @@ def __init__(self,
         :param train_filepath: the training data file path, e.g ``faiss.tgz`` or `faiss.npy`. The data file is expected
             to be either `.npy` file from `numpy.save()` or a `.tgz` file from `NumpyIndexer`.
         :param distance: 'l2' or 'inner_product' accepted. Determines which distances to optimize by FAISS
+        :param normalize: whether or not to normalize the vectors e.g. for the cosine similarity
```

Comment:
can u add a link to the documentation of faiss on how to achieve cosine similarity by normalizing?

Reply:
Of course, I will do that.